<?xml version="1.0"?>
<launch>
  <node pkg='convex_clustering_code' type='convex_clustering_code_node' name='convex_clustering_code_1' output="screen" >
    <remap from='input_pointcloud' to='velodyne_points_1' />
    <remap from='output_marker' to='output_marker_1' />
    <remap from='input_color_green' to='convex_clustering_code_1/color_green' />
    <remap from='input_color_red' to='convex_clustering_code_1/color_red' />
    <remap from='rviz_base' to='convex_clustering_code_1/velodyne_base' />

    <param name='color_green' value='0.5' />
    <param name='color_red' value='0' />
    <param name='velodyne_base' value='/velodyne_front_base_link' /> 
  </node>


  <node pkg='convex_clustering_code' type='convex_clustering_code_node' name='convex_clustering_code_2' output="screen" >
    <remap from='input_pointcloud' to='velodyne_points_2' />
    <remap from='output_marker' to='output_marker_2' />
    <remap from='input_color_green' to='convex_clustering_code_2/color_green' />
    <remap from='input_color_red' to='convex_clustering_code_2/color_red' />
    <remap from='rviz_base' to='convex_clustering_code_2/velodyne_base' />

    <param name='color_green' value='0' /> 
    <param name='color_red' value='0.5' />
    <param name='velodyne_base' value='/velodyne_rear_base_link' />

  </node>

</launch>